src/content/docs/apm/agents/php-agent/configuration/php-agent-configuration.mdx

```diff
@@ -1814,11 +1796,12 @@ The PHP agent can capture context data for the Monolog library and add its conte
         </tr>
         </tbody>
     </table>
-    If `context_data` is enabled for log forwarding, all attribute keys found in this list will be sent to New Relic in log records. If this list is empty, then all attributes will be sent.
 
-    <Callout variant="important">
-        When attribute rules are applied to log context attributes the "include" rules only include attributes which match, and drop attributes not covered by the "include" rules.  This is different than other types of agent attributes.  If log context attributes do not appear to be showing up as expected, be sure to check the "newrelic.attributes.include" ruleset to make sure it is not excluding the log context attribute.  If all "include" lists are empty then all log context attributes are forwarded.
-    </Callout>
+    When you apply attribute rules to log context attributes, the `include` rule drops all attributes not defined in the ruleset. This is a different behavior from other types of agent attributes. 
+    
+    * If log context attributes are not appearing as expected, then you should review the `newrelic.attributes.include` ruleset to see if your missing attributes are part of the exclusion. 
+    * If `context_data` is enabled for log forwarding, all attribute keys found in this list will appear in log records. An empty list will send all attribute keys as a default
```

Comment:
There is another troubleshooting tip we could include (needs some work to integrate into existing text):

- If log context attributes are not appearing as expected, first set `newrelic.loglevel=verbosedebug` to enable verbose log messages from the agent.  Then look for messages from the agent where it has excluded an attribute.  This can give clues to how to adjust the `include/exclude` rules.